README.markdown

```diff
@@ -33,6 +33,13 @@ To install:
 
 and you should be set.
 
+### Throubleshooting
+
+Mac users may face problems when are installing the package, it happens because on the instalation build, xcon tries to import a module that is provided by OpenSSL, and most of the cases it fails because it can't find the module.
```

Comment:
```suggestion
Mac users may face problems when installing the package, it happens because on the installation build, xcon tries to import a module that is provided by OpenSSL, and most of the cases it fails because it can't find the module.
```

`xcon`? :thinking: